README.md

```diff
@@ -10,13 +10,41 @@ cargo install stellar-contract-cli --git https://github.com/stellar/stellar-cont
 
 ## Usage
 
+All values passed to `--arg` are the JSON representation of SCVals.
+
+### Directly invoking a function in a user specified WASM contract
+
 ```
-stellar-contract-cli invoke --file <WASMFILE> --fn <FUNCNAME> --arg i32:1 --arg i32:2
+stellar-contract-cli invoke --file <WASMFILE> vm-fn --fn <FUNCNAME> --arg '{"i32":32}' --arg '{"i32":4}'
```

Comment:
Is the `vm-fn` still part of the invoke command?

Reply:
Nope. Removed.